ts/node/database_utility.ts

```diff
@@ -264,7 +264,7 @@ export function rebuildFtsTable(db: BetterSqlite3.Database) {
           CREATE TRIGGER messages_on_delete AFTER DELETE ON ${MESSAGES_TABLE} BEGIN
             DELETE FROM ${MESSAGES_FTS_TABLE} WHERE id = old.id;
           END;
-          CREATE TRIGGER messages_on_update AFTER UPDATE ON ${MESSAGES_TABLE} BEGIN
+          CREATE TRIGGER messages_on_update AFTER UPDATE ON ${MESSAGES_TABLE} WHEN new.body <> old.body BEGIN
```

Comment:
Just to confirm. Is this making sure that the search triggers have been updated only when the body of a message has changed?

Reply:
Not sure what you meant exactly, but yeah, it is meant to only rerun the triggers for the full text search logic when the body has changed and not anything else (i.e., marking a message as read does not update the body and so does not need to trigger a refresh of the full text search table)

Reply:
Yeah, I'm probably using the wrong terminology but you answered my question. Thank you 🙏